lib/graphql/schema/loader.rb

```diff
@@ -102,7 +102,7 @@ def define_type(type, type_resolver)
               name: type["name"],
               type: type_resolver.call(type["type"]),
               description: type["description"],
-              default_value: type["defaultValue"]
+              default_value: type["defaultValue"] ? JSON.parse(type["defaultValue"], quirks_mode: true) : nil
```

Comment:
😬  Glad to see this fixed! Is this change covered by a change in the tests? (As I review the diff, it's not jumping out at me.)